### PR TITLE
update ffi protocol to v0.4.7

### PR DESCRIPTION
**Pull request by cloudwebrtc**

### CODE REVIEW

File: Runtime/Scripts/Room.cs [hunk 1]
```diff
@@ -338,13 +340,33 @@ internal void OnEventReceived(RoomEvent e)
...
+                        {
+                            case DataPacketReceived.ValueOneofCase.None:
+                                //do nothing.
+                                break;
+                            case DataPacketReceived.ValueOneofCase.User:
+                                {
+                                    var dataInfo = e.DataPacketReceived.User;
+                                    var data = new byte[dataInfo.Data.Data.DataLen];
+                                    Marshal.Copy((IntPtr)dataInfo.Data.Data.DataPtr, data, 0, data.Length);
+                                    var participant = GetParticipant(e.DataPacketReceived.ParticipantSid);
```
Comment (davidzhao) [inline Runtime/Scripts/Room.cs:356]: I think we need to use identities here? we will deprecate SID from data messages eventually

Comment (cloudwebrtc) [reply Runtime/Scripts/Room.cs:356]: Yes, I am going to change all `ParticipantSid` to `ParticipantIdentity` for all other places in another PR

File: Runtime/Plugins/ffi-windows-arm64/livekit_ffi.dll
[the remark this answers is not in the document]
Comment (cloudwebrtc) [reply Runtime/Plugins/ffi-windows-arm64/livekit_ffi.dll:1]: I just checked that this file was converted from an ordinary file to LFS format, so it appears to have been deleted. When running `git clone` a normal-sized .dll file will be automatically generated.

[the remark this answers is not in the document]
Comment (cloudwebrtc) [reply Runtime/Plugins/ffi-windows-arm64/livekit_ffi.dll:1]: `-18.6 MB (0.00068%)`  It actually becomes a symbolic link, so its size is not completely reduced to 0.

**Review by davidzhao — APPROVED** [review]
sounds good.

**Pull request merged**